manifests/es-ruin-the-fun.yaml

```diff
@@ -0,0 +1,19 @@
+name: Ruin The Fun
+authors: Pshy
+homepage: https://github.com/pshy0/es-ruin-the-fun
+license: TFCL
```

Comment:
Is this a custom license? I can't find it in the [SPDX](https://spdx.org/licenses/) list.

Reply:
Yes it's custom, based on MIT.

Reply:
I changed the license to ISC, since ISC is actually more restrictive.

Reply:
Long-term we should really add a CUSTOM string, but for now this works.